resources/src/trade.rs

```diff
@@ -0,0 +1,253 @@
+use chrono::prelude::*;
+use amount::Amount;
+// use asset::AssetIdentifier;
+use offer::PriceRatio;
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+
+/// A trade represents an offer that was fulfilled between two assets and accounts.
+///
+/// https://www.stellar.org/developers/horizon/reference/resources/trade.html
+#[derive(Debug)]
+pub struct Trade {
+    id: String,
+    offer_id: String,
+    paging_token: String,
+    ledger_close_time: DateTime<Utc>,
+    base_account: String,
+    base_amount: Amount,
+    // base_asset: AssetIdentifier,
```

Comment:
?

Reply:
The assets have identifiers in the trade, i'm waiting on the "native" support in your branch.